recipes-kernel/linux/linux-riscv_4.15.bb

```diff
@@ -1,6 +1,9 @@
 require recipes-kernel/linux/linux-riscv-common.inc
 
+SUMMARY = "RISC-V Community BSP RISC-V Linux kernel with backported features and fiexes"
+DESCRIPTION = ""
```

Comment:
empty description is not preferred. Just do not set it, then summary appears as description as well.

Reply:
Dear @kraj.
Thanks to reply.
I PR it balnk to ask Desription content.

I suggest content below.
Below content is good or bad? If bad I will remove that Description field. 
```
DESCRIPTION = "Linux kernel based on mainline kernel used by RISC-V Community BSP in order to \
provide support for some backported features and fixes, or because it was applied in linux-next \
and takes some time to become part of a stable version, or because it is not applicable for \
upstreaming."
```

Reply:
looks good

Reply:
Thanks. I update PR.